Source/Core/Core/PrimeHack/Mods/ViewModifier.cpp

```diff
@@ -38,10 +39,17 @@ void ViewModifier::adjust_viewmodel(float fov, u32 arm_address, u32 znear_addres
         left = 0.22f;
         forward = -0.02f;
         apply_znear = true;
+        if (is_mp3_standalone_us) {
+          forward = 0.05f;
+          up = -0.36f;
+        }
       } else if (fov >= 75) {
         float factor = (fov - 75) / (125 - 75);
         left = Lerp(left, 0.22f, factor);
         forward = Lerp(forward, -0.02f, factor);
+        if (is_mp3_standalone_us) {
+          up = Lerp(up, -0.48f, factor);
+        }
```

Comment:
What is the reason for altering up/down and adjusting forward? Theoretically this should be exactly the same for all versions of Prime 3.

Reply:
For some unknown reasons it wouldn't adjust well on US Standalone version